src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -598,6 +599,7 @@ public sealed override ImmutableArray<FieldSymbol> TupleElements
                             var field = (FieldSymbol)member;
                             if (field is TupleVirtualElementFieldSymbol)
                             {
+                                // We will re-add all necessary virtual element field symbols below.
```

Comment:
>// We will re-add all necessary virtual element field symbols below. [](start = 32, length = 68)

It feel like it would be good to explain/give examples of scenarios when ```currentMembers``` already includes a field like that.